entry_types/scrolled/app/views/pageflow_scrolled/entries/show.html.erb

```diff
@@ -15,6 +15,10 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
 
+    <script>
+      pageflowScrolledSeed = <%= scrolled_entry_json_seed(@entry) %>;
+    </script>
+
```

Comment:
This currently makes some editor specs for scrolled fail, since [the template is also rendered as an HTML seed](https://github.com/codevise/pageflow/blob/d3c62f8fddce1c0f2d919c5d4a2a87e40e2facf9/entry_types/scrolled/app/helpers/pageflow_scrolled/editor/seed_html_helper.rb#L6) for the editor iframe. `@entry` is not set in this context. Maybe we can default to setting `pageflowScrolledSeed` to an empty array in that case. Then we can tackle displaying real data in the editor as a separate step.